platform/mbed-trace/include/mbed-trace/mbed_trace.h

```diff
@@ -70,7 +70,7 @@ extern "C" {
 #endif
 
 #ifndef MBED_CONF_MBED_TRACE_FEA_IPV6
-#define MBED_CONF_MBED_TRACE_FEA_IPV6 1
+#define MBED_CONF_MBED_TRACE_FEA_IPV6 MBED_CONF_NANOSTACK_LIBSERVICE_PRESENT
```

Comment:
This will fail if nanostack-libservice is unavailable - undefined macro. I'll push a fix.

Reply:
Fixed